assets/app/scripts/services/data.js

```diff
@@ -483,11 +469,25 @@ angular.module('openshiftConsole')
     }
   };
 
-  var URL_ROOT_TEMPLATE = "{protocol}://{+serverUrl}/{apiRoot}/{apiVersion}/";
+  var URL_ROOT_TEMPLATE = "{protocol}://{+serverUrl}{+apiPrefix}/";
```

Comment:
Do we want to be told the api version, or have the UI know the version of the API it was coded against? Seems like the UI should get to stay on a k8s and openshift API version until it is ready to move up